lms/models/assignment.py

```diff
@@ -98,11 +111,12 @@ class Assignment(CreatedUpdatedMixin, Base):
         MutableDict.as_mutable(JSONB()),
         server_default=sa.text("'{}'::jsonb"),
         nullable=False,
+        repr=False,
     )
 
     is_gradable: Mapped[bool] = mapped_column(
         sa.Boolean(),
-        default=False,
+        insert_default=False,
```

Comment:
default is now about data class default vs the existing behaviour now on `insert_default`.